packages/chrome/src/elements/nav/Nav.tsx

```diff
@@ -25,8 +25,8 @@ export const NavComponent = React.forwardRef<HTMLElement, INavProps>(
     return (
       <ThemeProvider
         theme={parentTheme => ({
-          ...parentTheme,
-          colors: { ...parentTheme.colors, base: isLight ? 'light' : 'dark' }
+          ...parentTheme!,
+          colors: { ...parentTheme!.colors, base: isLight ? 'light' : 'dark' }
```

Comment:
```suggestion
          ...parentTheme,
          colors: { ...parentTheme.colors, base: isLight ? 'light' : 'dark' }
```

Change the `ThemeProvider` import to `@zendeskgarden/react-theming`.